web/run.py

```diff
@@ -91,6 +91,14 @@ def check_if_token_is_revoked(decrypted_token):
     login_manager.login_message = "You must be logged in to access this page!!!"
     login_manager.login_view = "auth.login"
 
+    # CORS
+    @app.after_request
+    def apply_caching(response):
+        reqURL = request.base_url
+        if config.getCORS() and reqURL.count("/api/") == 1 and reqURL.count("/whitelist") == 0 and reqURL.count("/blacklist") == 0:
```

Comment:
I believe `and reqURL.count("/whitelist") == 0 and reqURL.count("/blacklist") == 0` could be simplified as `and reqURL.count("/admin") == 0` as all the administrative paths do have `/admin` at the beginning, right?

Reply:
You are absolutely right

Reply:
And with Black formatting:
```
-        if config.getCORS() and reqURL.count("/api/") == 1 and reqURL.count("/whitelist") == 0 and reqURL.count("/blacklist") == 0:
-            response.headers.add('Access-Control-Allow-Origin', config.getCORSAllowOrigin())
+        if (
+            config.getCORS()
+            and reqURL.count("/api/") == 1
+            and reqURL.count("/admin") == 0
+        ):
+            response.headers.add(
+                "Access-Control-Allow-Origin", config.getCORSAllowOrigin()
+            )
```

...better use automated formatter, though:

```
black web/run.py
black lib/Config.py
```